susemanager-utils/susemanager-sls/src/grains/cpuinfo.py

```diff
@@ -139,9 +139,10 @@ def cpu_data():
                     "Core(s) per socket": "cpu_cores",
                 }
                 values = {}
-                for entry in data.get("lscpu"):
-                    if entry["field"][:-1] in name_map.keys():
-                        values[name_map[entry["field"][:-1]]] = entry["data"]
+                for line in lines:
+                    parts = [l.strip() for l in line.split(":")]
```

Comment:
Maybe this way in case if value could contain `:`?
```suggestion
                    parts = [l.strip() for l in line.split(":", 1)]
```